.github/workflows/if-nodejs-pr-testing.yml

```diff
@@ -72,3 +72,20 @@ jobs:
         name: Run release assets generation to make sure PR does not break it
         shell: bash
         run: npm run generate:assets --if-present
+
+  upload-coverage:
+    name: Upload Coverage to Codecov
+    runs-on: ubuntu-latest
+    needs: test-nodejs-pr
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v4
+      - name: Upload coverage to Codecov
+        uses: codecov/codecov-action@v4
+        with:
+          fail_ci_if_error: true
+          files: ./coverage/lcov.info
+          flags: unittests
+          name: codecov-umbrella
+          token: ${{ secrets.CODECOV_TOKEN }}
```

Comment:
I think we can explicitly mention the token here. 
Ref: https://github.com/jaegertracing/jaeger-ui/blob/3b093f81dec59c6bb04daad96b1b77bd03a29e4a/.github/workflows/unit-tests.yml#L34-L36

Reply:
My token is configured for the forked repo, but we need to obtain a token for the original repo. This can only be done by signing up for Codecov using the organization's GitHub account, and I believe only Lukasz can do that. Maybe, before this PR is merged, he can add the token directly here or to the script. I've already asked him to have a look at this PR.

Reply:
tbh I don't know how to install it, is it a GH app that I should install or? I installed it for website repo, but where to get token from?

Reply:
I am still curious if we should make the token public or add it as a GitHub token
@derberg @akshatnema ?

Why? -> https://github.com/jaegertracing/jaeger-ui/blob/3b093f81dec59c6bb04daad96b1b77bd03a29e4a/.github/workflows/unit-tests.yml#L34-L36